apps/web/src/components/PoolCandidatesTable/PoolCandidateFilterDialog.tsx

```diff
@@ -0,0 +1,317 @@
+            label: `${group}-0${level}`,
+          }))}
+        />
+        <Combobox
+          id="stream"
+          name="stream"
+          isMulti
+          label={intl.formatMessage(adminMessages.streams)}
+          options={enumToOptions(PoolStream).map(({ value }) => ({
+            value,
+            label: intl.formatMessage(getPoolStream(value)),
+          }))}
+        />
+        <Combobox
+          id="poolCandidateStatus"
+          name="poolCandidateStatus"
+          isMulti
+          label={intl.formatMessage(adminMessages.status)}
+          options={enumToOptions(PoolCandidateStatus).map(({ value }) => ({
+            value,
+            label: intl.formatMessage(getPoolCandidateStatus(value)),
+          }))}
+        />
+        <Checklist
+          idPrefix="operationalRequirement"
+          name="operationalRequirement"
+          legend={intl.formatMessage(navigationMessages.workPreferences)}
+          items={OperationalRequirementV2.map((value) => ({
+            value,
+            label: intl.formatMessage(
+              getOperationalRequirement(value, "short"),
+            ),
+          }))}
+        />
+        <Checklist
+          idPrefix="workRegion"
+          name="workRegion"
+          legend={intl.formatMessage(navigationMessages.workLocation)}
+          items={enumToOptionsWorkRegionSorted(WorkRegion).map(({ value }) => ({
+            value,
+            label: intl.formatMessage(getWorkRegion(value)),
+          }))}
+        />
+        <div
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(column)"
+          data-h2-gap="base(x1 0)"
+        >
+          <RadioGroup
+            idPrefix="expiryStatus"
+            name="expiryStatus"
+            legend={intl.formatMessage({
+              defaultMessage: "Expiry status",
+              description: "Label for the expiry status field",
+              id: "HDiUEc",
+            })}
+            items={enumToOptions(CandidateExpiryFilter).map(({ value }) => ({
+              value,
+              label: intl.formatMessage(getCandidateExpiryFilterStatus(value)),
+            }))}
+          />
+          <RadioGroup
+            idPrefix="suspendedStatus"
+            name="suspendedStatus"
+            legend={intl.formatMessage({
+              defaultMessage: "Candidacy status",
+              description: "Label for the candidacy status field",
+              id: "NxrKpM",
+            })}
+            items={enumToOptions(CandidateSuspendedFilter).map(({ value }) => ({
+              value,
+              label: intl.formatMessage(
+                getCandidateSuspendedFilterStatus(value),
+              ),
+            }))}
+          />
+        </div>
+        <div
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(column)"
+          data-h2-gap="base(x.5)"
+          data-h2-align-self="l-tablet(center)"
+        >
+          <Checkbox
```

Comment:
Oops, this is being sent as a null.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/e8940845-9501-4cfb-a42b-61fd250f3730)

Reply:
Looks like this is yet another filter that is purposely being left out of the query 😞 I'll try to remember to bring it up during stand up to see what we want to do.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/daf6a72d8a978b94fc140ab865c9e6c14d20be68/apps/web/src/components/PoolCandidatesTable/PoolCandidatesTable.tsx#L299